lxd/storage_migration.go

```diff
@@ -41,6 +41,7 @@ func (s rsyncStorageSourceDriver) Snapshots() []container {
 }
 
 func (s rsyncStorageSourceDriver) SendWhileRunning(conn *websocket.Conn, op *operation, bwlimit string) error {
+	ctName, _, _ := containerGetParentAndSnapshotName(s.container.Name())
```

Comment:
Mostly a nitpick but you could pass s.container as parameter, and have containerGetParentAndSnapshotName call call .Name() on it